### PR TITLE
par2: Move to the 0.8.1 version in a maintained clone.

### PR DESCRIPTION
**Pull request by wteiken**
#### Description

The par2 packet is from a stale repo and does not compile on arm due to issues with config.guess.  This PR replaces the package source with a maintained clone of the original that compiles cleanly.

###### Type(s)

- [x] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 12.0.1, arm64
Xcode 13.1

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: archivers/par2/Portfile [hunk 1]
```diff
@@ -2,8 +2,8 @@ PortSystem        1.0
 PortGroup         compiler_blacklist_versions 1.0
 
 name              par2
-version           0.4
-revision          4
+version           0.8.1
+revision          1
```
Comment (reneeotten) [inline archivers/par2/Portfile:6]: the "revision" field should be reset to zero when updating the "version"

File: archivers/par2/Portfile [hunk 2]
```diff
@@ -17,20 +17,18 @@ license           GPL-2
 maintainers       jabber.org:julian
 categories        archivers
 platforms         darwin 
-homepage          http://parchive.sourceforge.net/
-master_sites      sourceforge:parchive
+homepage          https://github.com/Parchive/par2cmdline
+master_sites      ${homepage}/releases/download/v${version}
```
Comment (reneeotten) [inline archivers/par2/Portfile:21]: this should use the "github" PortGroup; several fields are set then correctly automatically.

**Review by reneeotten — CHANGES_REQUESTED** [review]
thanks @wteiken - I have no opinion on the change to a fork; but if this is do e I've left a few comments.

File: archivers/par2/Portfile (hunk 2 above)
Comment (wteiken) [reply archivers/par2/Portfile]: Done

File: archivers/par2/Portfile (hunk 1 above)
Comment (wteiken) [reply archivers/par2/Portfile]: Done

File: archivers/par2/Portfile [hunk 3]
```diff
@@ -1,9 +1,10 @@
 PortSystem        1.0
 PortGroup         compiler_blacklist_versions 1.0
+PortGroup         github 1.0
 
 name              par2
-version           0.4
-revision          4
+github.setup      Parchive par2cmdline v0.8.1
```
Comment (reneeotten) [inline archivers/par2/Portfile:6]: the "v" should not be part of the version, it should be "0.8.1 v" instead

File: archivers/par2/Portfile [hunk 4]
```diff
@@ -17,20 +18,18 @@ license           GPL-2
 maintainers       jabber.org:julian
 categories        archivers
 platforms         darwin 
-homepage          http://parchive.sourceforge.net/
-master_sites      sourceforge:parchive
+homepage          https://github.com/Parchive/par2cmdline
```
Comment (reneeotten) [inline archivers/par2/Portfile:21]: the homepage is set automatically by the github PG so you can rove this line

File: archivers/par2/Portfile [hunk 5]
```diff
@@ -17,20 +18,18 @@ license           GPL-2
...
-                  sha1 2fcdc932b5d7b4b1c68c4a4ca855ca913d464d2f \
-                  rmd160 e98dc84fa7a4304b8d71c1d373fdce0bca1e6290
+checksums         sha1 96713067eb29ef18aba64c5c168c64d6f72c8300 \
+                  rmd160 5e9bb8fd1b71c41c7b3fe1adf729df7b8f7936bd \
+                  sha256 7b2fcc19b54c7080939fc2cbaca33ae40ac33759a526292933c94a85ba850d11 \
+                  size 3654317
 
-distname          par2cmdline-${version}
+github.tarball_from releases
+distname          ${github.project}-[strsed ${github.version} {s/v//}]
```
Comment (reneeotten) [inline archivers/par2/Portfile:29]: distname should be set automatically

**Review by reneeotten — CHANGES_REQUESTED** [review]
a few more things to take care of

File: archivers/par2/Portfile (hunk 4 above)
Comment (reneeotten) [reply archivers/par2/Portfile:21]: please also fix the lint warning: `Line 20 has trailing whitespace before newline`

File: archivers/par2/Portfile [hunk 6]
```diff
@@ -17,20 +18,18 @@ license           GPL-2
...
 categories        archivers
 platforms         darwin 
-homepage          http://parchive.sourceforge.net/
-master_sites      sourceforge:parchive
+homepage          https://github.com/Parchive/par2cmdline
 
-checksums         md5 1551b63e57e3c232254dc62073b723a9 \
-                  sha1 2fcdc932b5d7b4b1c68c4a4ca855ca913d464d2f \
-                  rmd160 e98dc84fa7a4304b8d71c1d373fdce0bca1e6290
+checksums         sha1 96713067eb29ef18aba64c5c168c64d6f72c8300 \
```
Comment (reneeotten) [inline archivers/par2/Portfile:23]: `sha1` checksum can be dropped, we don't use that anymore

File: archivers/par2/Portfile (hunk 3 above)
Comment (wteiken) [reply archivers/par2/Portfile]: Ah, that is what I missed. Thx! Done.

File: archivers/par2/Portfile (hunk 4 above)
Comment (wteiken) [reply archivers/par2/Portfile]: Done.

File: archivers/par2/Portfile (hunk 5 above)
Comment (wteiken) [reply archivers/par2/Portfile]: Done.

File: archivers/par2/Portfile (hunk 6 above)
Comment (wteiken) [reply archivers/par2/Portfile]: Done.

**Comment by wteiken** [comment]
Friendly ping, what is the next step here?

**Comment by pmetzger** [comment]
@reneeotten ?

**Comment by ra1nb0w** [comment]
All CIs are good. Merge. thank you.

**Pull request merged**